erica/elster_xml/common/basic_xml_data_representation.py

```diff
@@ -0,0 +1,85 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class ENutzdaten:
+    """ Subclass this for each Use Case """
+    pass
+
+
+@dataclass
+class EEmpfaenger:
```

Comment:
> One more thing: Maybe we could also order the classes from top to bottom. So that I see a class used in another one before it is defined (or is that not possible because of referencing?)

That's not possible for precisely that reason. :)

> Personally, I think more doccomments or comments in general would help a lot in this file to explain what the different classes are, what the attributes are, what names can be changed...

Do you mean for each single class? Probably not?

> And maybe also a general comment which explains the part the xml consists of (Nutzdatenheader etc.) or a reference to a schema file that explains that?

I'll add that and also will try to explain what you asked for in the aforementioned (english - yeah!) comment^^